frontend/views/containers/payments/PaymentsMixin.js

```diff
@@ -0,0 +1,97 @@
+import sbp from '@sbp/sbp'
+import { mapState, mapGetters } from 'vuex'
+import { PAYMENT_COMPLETED } from '@model/contracts/shared/payments/index.js'
+import { simplifyPayment, getPaymentHashes } from '@model/contracts/shared/functions.js'
+import { cloneDeep } from '@model/contracts/shared/giLodash.js'
+
+const PaymentsMixin: Object = {
```

Comment:
I would add a comment above this explaining that this file combines payment information from both the current in-memory state and the archived payments on disk.